frontend/src/app/shared/dropdown/dropdown.component.ts

```diff
@@ -65,6 +74,14 @@ export class DropdownComponent<T> implements OnChanges, AfterViewInit, OnDestroy
         return this.dropdownService.open$.value;
     }
 
+    get triggerID(): string {
+        return `dropdown-trigger-${this.id}`;
```

Comment:
Could we make the ids more semantic by adding `labelledBy` content instead?

Reply:
Yes, the button will be named `dropdown-trigger-4` or such, based on the order of construction.

You could allow more descriptive IDs, but since IDs are not visible to the user (unless they inspect the HTML), I don't think that would really have any benefit.